scripts/git-log-pretty-since.sh

```diff
@@ -40,13 +40,13 @@ _format_ticketnums() {
 }
 
 for commit in $(git log --format='%H' --first-parent "$__SINCE"..HEAD); do
-  if is_merge_commit ${commit} ; then
-      if [ -z "$__INVMATCH" ] || ! git log --oneline ${commit}^1..${commit}^2 \
+  if is_merge_commit "${commit}" ; then
+      if [ -z "$__INVMATCH" ] || ! git log --oneline "${commit}"^1.."${commit}"^2 \
```

Comment:
nit (similar cases below): 

```suggestion
      if [ -z "$__INVMATCH" ] || ! git log --oneline "${commit}^1..${commit}^2" \
```